CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing to Common Voice
+
+🎉 First off, thanks for taking the time to contribute! This project would not be possible without people like you. 🎉
+
+## Code of Conduct
+By participating in this project, you're agreeing to uphold the [Mozilla Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
+
+## How Can I Contribute?
+
+### Non-code
+Help us add more sentences for other volunteers to read. See [issue 341](https://github.com/mozilla/voice-web/issues/341) for details.
+
+### Bug fixes and feature enhancements
+All of our current issues can be found here on GitHub. Anything with a [help wanted](https://github.com/mozilla/voice-web/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) tag is up for grabs.
+
+#### Requirements
+- [NodeJS](https://nodejs.org/en/) (v7 or higher)
+- [npm](https://www.npmjs.com/) (4 or higher)
```

Comment:
@mikehenrty, should we just switch the official requirement to Node 8 & npm 5 right here?

Reply:
we are still using node 7 in travis. what would be the advantage of switching?

Reply:
hm. you're right. in light of us likely switching to yarn ( @carolstran, don't worry, we'll update this when we actually do switch), requiring NPM 5 doesn't give us much.

Let's leave as is for now. If we have other reasons for requiring Node 8, we can update this at that time.

@mikehenrty, as an aside, travis build works fine with Node 8 as well (I tried it a little while ago in my fork)